### ISSUE TITLE
append_row moving data the the right

### ISSUE THREAD

**Issue by fluffyduffdylan** [burnash/gspread#603]
### Environment info

Operating System: Windows 10
Python version: 3.4.3
gspread version: 3.0.1

### Steps to reproduce
Sometimes appending two rows consecutively will cause the second one to be shifted a few columns to the right. Here's how to reproduce:
`>>> import gspread`
`>>> from oauth2client.service_account import ServiceAccountCredentials`
`>>> import json`
`>>> json_key = json.load(open("client_secret.json"))`
`>>> scope = ["https://spreadsheets.google.com/feeds", "https://www.googleapis.com/auth/drive"]`
`>>> creds = ServiceAccountCredentials.from_json_keyfile_name('client_secret.json', scope)`
`>>> client = gspread.authorize(creds)`
`>>> new_project = client.create("test_project")`
`>>> new_project.share(email, perm_type="user", role="owner")`
`>>> sheet1 = new_project.worksheet("Sheet1")`
`>>> sheet1.append_row(["Activity Number", "Activity Name", "Activity Time", "", "", "Number of", "", "Number of", "", "Number of Y12"])`
`{'updates': {'updatedRange': 'Sheet1!A1:J1', 'updatedColumns': 10, 'updatedRows': 1, 'spreadsheetId': '15NqyDlPeZ_aS0F9WVvrZfyY4wrJ-nC7Oy-Wh0g86J74', 'updatedCells': 10}, 'spreadsheetId': '15NqyDlPeZ_aS0F9WVvrZfyY4wrJ-nC7Oy-Wh0g86J74'}`
`>>> sheet1.append_row(["", "", "Morning", "Afternoon", "Evening", "Min", "Max", "Min", "Max", "Min", "Max"])`
`{'tableRange': 'Sheet1!J1', 'spreadsheetId': '15NqyDlPeZ_aS0F9WVvrZfyY4wrJ-nC7Oy-Wh0g86J74', 'updates': {'updatedRange': 'Sheet1!J2:T2', 'updatedColumns': 11, 'updatedRows': 1, 'spreadsheetId': '15NqyDlPeZ_aS0F9WVvrZfyY4wrJ-nC7Oy-Wh0g86J74', 'updatedCells': 11}}`

### Stack trace or other output that would be helpful

This is what the above code produces in Google Sheets

Activity Number | Activity Name | Activity Time |   |   | Number of |   | Number of |   | Number of Y12 |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  |   |   |   |   |   |   |   |   |   |   | Morning | Afternoon | Evening | Min | Max | Min | Max | Min | Max

**Comment by te-evalenzuela** [comment]
Can you see if this continues happening for you if you specify the value_input_option parameter as 'USER_ENTERED'?
e.g. >>> sheet1.append_row(['foo'], 'USER_ENTERED')
ref: https://developers.google.com/sheets/api/reference/rest/v4/ValueInputOption
I've had strange things happen with 'RAW' occasionally.

If that doesn't work, you might want to try adding 'majorDimension' : 'ROWS' to the "body" dictionary in models.append_rows().
i.e.
> body = {
        'majorDimension': 'ROWS',
        'values': values
}

**Comment by giuliocandre** [comment]
Same problem here, it doesn't work even with 'USER_ENTERED' and 

> 'majorDimension': 'ROWS',

as suggested by @te-evalenzuela 

Any other hints on how to solve this?

**Comment by Logic-gate** [comment]
I had a similar issue, ended up writing a helper function,

it adds each cell independently; not efficient, bit it does serve the purpose.   

```
....
...
..

def update_new_row_entry(self, *value):
    	"""
    		TODO: during testing all_records showed extra fields, check later
		Update first empty row with values

		:params value: String cell update values
    	"""
    	new_row = len(self.all_records()) + 2
    	for i, ii in zip(range(1, len(value) + 1), value):
    		self.sheet.update_cell(new_row, i, ii)

```
where `all_records` is `get_all_records()`

**Comment by lavigne958** [comment]
This relates to #537  and has been solved by #726 .

One can now provide the starting range where to insert the new row.

Closing this issue.
